### PR TITLE
Reference catalogue for open standards

### PR DESCRIPTION
**Pull request by prajectory**
Fixes # .

## Changes proposed in this pull request:
- 
-
-

## Propagation of Changes

Identify where of the following the proposed changes need to be propagated, and include the relevant pull request where appropriate:

- [ ] The DPGA website contains https://digitalpublicgoods.net/standard/ which needs to be updated manually matching the contents of [standard.md](https://github.com/DPGAlliance/DPG-Standard/blob/master/standard.md).
- [ ] The DPGA website also contains the [submission guide](https://digitalpublicgoods.net/submission-guide/) which needs to be updated manually.
- [ ] The [unicef/publicgoods-candidates](https://github.com/unicef/publicgoods-candidates) repository needs the following to be updated:
    - [ ] the [nominee-schema.json](https://github.com/unicef/publicgoods-candidates/blob/master/nominee-schema.json)
    - [ ] the [screening-schema.json](https://github.com/unicef/publicgoods-candidates/blob/master/screening-schema.json)
    - [ ] propagate any changes to all encoded nominees and digital public goods.
    - Link to the corresponding pull request: unicef/publicgoods-candidates#
- [ ] The [Submission Form](https://digitalpublicgoods.net/submission) through the [unicef/publicgoods-submission](https://github.com/unicef/publicgoods-submission) repository, and its corresponding [schema.js](https://github.com/lacabra/submission-digitalpublicgoods/blob/master/schemas/schema.js)
    - Link to the corresponding pull request:
- [ ] The [Eligibility Form](https://digitalpublicgoods.net/eligibility/) through the [unicef/publicgoods-scripts](https://github.com/unicef/publicgoods-submission) repository, by editing [quizQuestions.js](https://github.com/unicef/publicgoods-scripts/blob/master/packages/eligibility/src/api/quizQuestions.js)
    - Link to the corresponding pull request:

### CODE REVIEW

File: DPG reference catalog for open standards [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+XML
+
+#Character encoding and languages:
+UTF-8, ISO/IEC 10646
+ISO 639-1/639-2/639-3
+
+#Web standards:
+HTML 
+CSS
+SSL
```
Comment (downeymj) [inline DPG reference catalog for open standards:15]: SSL (even SSLv3) is no longer recommended due to security vulnerabilities, and has generally been replaced with TLS v1.2 or v1.3. More information: https://wiki.mozilla.org/Security/Server_Side_TLS

File: DPG reference catalog for open standards [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+Codebase:
+Public Code
+FAIR Guiding Principles for scientific data management and stewardship
+
+#Design: 
+SDLC
+SSDLC
+
+#Information security: 
+OWASP/ ASVS,
```
Comment (downeymj) [inline DPG reference catalog for open standards:52]: Any specific version of the ASVS that people should use? Is there a specific Level within the ASVS that people should be aiming for with respect to their project(s)? Should complementary guides such as the (mobile) MASVS or OWASP's IoT evaluation framework also be mentioned now that they are being split out from the main ASVS?

**Review by downeymj — CHANGES_REQUESTED** [review]
FYI, this file does not currently seem to have an extension, e.g., `.md`.

File: DPG reference catalog for open standards [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+UTF-8, ISO/IEC 10646
+ISO 639-1/639-2/639-3
+
+#Web standards:
+HTML 
+CSS
+SSL
+HTTP
+XHTML (eXtensible HTML v1.0)
+FTP
```
Comment (downeymj) [inline DPG reference catalog for open standards:18]: The use of FTP (vs. more secure protocols like HTTPS or SFTP) is generally discouraged in technical communities due to its age and security status, and it is noteworthy that its support was removed from most browsers in 2020. More info: https://groups.google.com/g/mozilla.dev.platform/c/FqCZUT9ay_o/m/jt4DLRDjAwAJ

File: DPG reference catalog for open standards [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+#Character encoding and languages:
+UTF-8, ISO/IEC 10646
+ISO 639-1/639-2/639-3
+
+#Web standards:
+HTML 
+CSS
+SSL
+HTTP
```
Comment (downeymj) [inline DPG reference catalog for open standards:16]: Related to the SSL comment, browsers are starting to offer HTTPS-only modes and with the rise of readily-available certificates through services such as Lets Encrypt, the use of non-encrypted HTTP is becoming more and more discouraged. https://https.cio.gov/everything/

Comment (lacabra) [reply DPG reference catalog for open standards:16]: FYI @nathanbaleeta

File: DPG reference catalog for open standards (hunk 3 above)
Comment (lacabra) [reply DPG reference catalog for open standards:18]: Agreed. FYI @nathanbaleeta

File: DPG reference catalog for open standards (hunk 2 above)
Comment (lacabra) [reply DPG reference catalog for open standards:52]: FYI @nathanbaleeta

**Pull request closed without merging**